src/main/kotlin/extensions/ExtensionsCollector.kt

```diff
@@ -12,6 +13,7 @@ object ExtensionsCollector {
      */
     val extensions = listOf(
             PropertyPrefixSupport,
-            PropertySuffixSupport
+            PropertySuffixSupport,
+            PropertyAnnotationLineSupport
```

Comment:
Could we make the name to 'PropertyAnnotationSingleLineSupport?'

Reply:
> Could we make the name to 'PropertyAnnotationSingleLineSupport?'

Totally we can. but why I named it without "single" is that if a property with multiple annotation, maybe it's not a good way to read in single line.